addons/survey/static/src/views/survey_views.js

```diff
@@ -0,0 +1,81 @@
+/** @odoo-module **/
+
+import { registry } from '@web/core/registry';
+import { ListRenderer } from "@web/views/list/list_renderer";
+import { KanbanRenderer } from "@web/views/kanban/kanban_renderer";
+import { listView } from '@web/views/list/list_view';
+import { kanbanView } from '@web/views/kanban/kanban_view';
+import { useService } from "@web/core/utils/hooks";
+import { patch } from "@web/core/utils/patch";
+
+const { useEffect, useRef } = owl;
+
+const SurveySampleMixin = {
+    useSurveyLoadSampleHook(selector) {
+        const rootRef = useRef("root");
+        useEffect(
+            (elems) => {
+                if (!elems || !elems.length) {
+                    return;
+                }
+                const handler = (ev) => {
+                    const surveyMethod = ev.currentTarget.closest('.o_survey_sample_container').getAttribute('action');
+                    this.loadSample(surveyMethod);
+                }
+                for (const elem of [...elems]) {
+                    elem.addEventListener('click', handler);
+                }
+                return () => {
+                    for (const elem of [...elems]) {
+                        elem.removeEventListener('click', handler);
+                    }
+                };
+            },
+            () => [rootRef.el && rootRef.el.querySelectorAll(selector)]
+        );
+    },
+
+    setup() {
+        this._super();
+        this.actionService = useService("action");
+        this.orm = useService('orm');
+        this.isLoadingSample = false;
+        this.useSurveyLoadSampleHook('.o_survey_load_sample');
+    },
+
+    /**
+     * Load and show the sample survey related to the clicked element,
+     * when there is no survey to display.
+     * We currently have 3 different samples to load:
+     * - Sample Feedback Form
+     * - Sample Certification
+     * - Sample Live Presentation
+     */
+    async loadSample(method) {
+        // Prevent loading multiple samples if double clicked
+        const canCreate = this.props.archInfo.activeActions.create;
```

Comment:
Actually this should never change, so we should call the hook only if create is true, and not check it in this function.